### ISSUE TITLE
Commit https://github.com/Qualys/log4jscanwin/commit/d7b27613f717027e244498892cc530f754560c0c breaks the /lowpriority switch on 64-bit systems. 

### ISSUE THREAD

**Issue by nagten** [Qualys/log4jscanwin#36]
Commit https://github.com/Qualys/log4jscanwin/commit/d7b27613f717027e244498892cc530f754560c0c breaks the /lowpriority switch on 64-bit systems. 

With versions 2.0.2.4 the Log4jscanner process is running under below normal CPU priority which is fine but I/O priority is normal (you can check easily with Sysinternals Process Explorer) on 64-bit systems. Before this change CPU was running with low priority and I/O with very low on 64-bit systems. 

I see this change was made to support 32-bit systems. Currently every 32-bit Windows Server operating system is out of support but Win7 and 32-bit Windows 10 is still supported. Pull request #37 fixes this.

I'm also sure reports won't be written on some 32-bit operating systems (Windows 2003 and WinXP) because system variable %PROGRAMDATA% is not supported/available on these operating systems https://github.com/Qualys/log4jscanwin/blob/1b48c17773d23220df6928c2c405bf358c1c4009/Utils.h#L15

**Comment by romw** [comment]
> I'm also sure reports won't be written on some 32-bit operating systems (Windows 2003 and WinXP) because system variable %PROGRAMDATA% is not supported/available on these operating systems
> 
> https://github.com/Qualys/log4jscanwin/blob/1b48c17773d23220df6928c2c405bf358c1c4009/Utils.h#L15

I think I'm going to wait on separate bug report before I begin worry about this issue too much.

**Comment by romw** [comment]
Thanks a bunch for the rest of the submission.
